em-metadata/__init__.py

```diff
@@ -0,0 +1,25 @@
+# **************************************************************************
+# *
+# * Authors:  J. M. de la Rosa Trevin (delarosatrevin@gmail.com)
```

Comment:
A guy named Grigory Sharov has made some contributions, we should add him as an author here.